docs/actions/builtin-serve-static-files.part.md

```diff
@@ -29,3 +11,7 @@ Pay attention this action is only available if your objectstore is configured to
 * View your static files with links in the tab artifact
 
 ![img](/images/workflows.pipelines.actions.builtin.serve-static-files-tab.png)
+
+## Notes
+
+Pay attention this action is only available if your objectstore is configured to use Openstack Swift. And fow now by default your static files will be deleted after 2 months.
```

Comment:
Maybe also available for S3 now ? @yesnault ?

Reply:
yes